src/sidebar/store/modules/session.ts

```diff
@@ -92,6 +124,9 @@ function hasFetchedProfile(state: State) {
  *
  * If the profile has not yet been fetched yet, a dummy logged-out profile is
  * returned. This allows code to skip a null check.
+ *
+ * NOTE: To check the set of enabled features, use the {@link features} selector
+ * instead, since it combines features from the profile and other sources.
```

Comment:
Maybe we should also mark `features` as deprecated in `Profile` type.

Reply:
The `Profile` type is the object we get back from the API. As such the features property isn't deprecated, code in the client needs to be aware that it should use `store.features()` rather than `store.profile().features`. We could potentially enforce this in feature by making `store.profile()` return an object which is different than the API response, and doesn't have a `features` field on it.